components/script/dom/element.rs

```diff
@@ -2214,6 +2214,7 @@ impl VirtualMethods for Element {
         match attr.local_name() {
             &local_name!("style") => {
                 // Modifying the `style` attribute might change style.
+                node.dirty(NodeDamage::NodeStyleDamaged);
```

Comment:
This shouldn't be needed. We handle that already here: https://github.com/servo/servo/blob/master/components/script/dom/document.rs#L2418.